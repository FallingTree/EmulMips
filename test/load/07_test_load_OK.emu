#-------
#TEST_RETURN_CODE=PASS          
#TEST_COMMENT="Test d'ouverture d'un fichier"
#-------

load procedure.o 0x0000AAAA
