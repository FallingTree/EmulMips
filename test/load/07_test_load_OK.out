Bienvenue dans l'Emulateur MIPS !
Charger un fichier elf
La commande s'est exécutée correctement

Charger un fichier elf
La commande s'est exécutée correctement

Charger un fichier elf
La commande s'est exécutée correctement

Charger un fichier elf
La commande s'est exécutée correctement

Charger un fichier elf
La commande s'est exécutée correctement

Charger un fichier elf
Réinitialisation des registres, suppression des breakpoints
La commande s'est exécutée correctement

Réinitialisation des registres, suppression des breakpoints
La commande s'est exécutée correctement

.reginfo 
  0 :: 20090008    ADDI $t1 $zero 8
  4 :: 00095020    ADD $t2 $zero $t1
  8 :: 00094880    SLL $t1 $t1 2
  c :: 0c000009    JAL 0x24
  10 :: 00000000    NOP
  14 :: 112a0005    BEQ $t1 $t2 5
  18 :: 00000000    NOP
  1c :: 0800000c    J 0x30
  20 :: 00000000    NOP
  24 :: 00094882    SRL $t1 $t1 2
  28 :: 03e00008    JR $ra
  2c :: 00000000    NOP
  30 :: 00000000    NOP
La commande s'est exécutée correctement

